process/elasticproc/block/blockProcessor_test.go

```diff
@@ -274,3 +288,132 @@ func TestBlockProcessor_PrepareBlockForDBEpochStartMeta(t *testing.T) {
 		DeveloperFees:     "0",
 	}, dbBlock)
 }
+
+func TestBlockProcessor_PrepareBlockForDBMiniBlocksDetails(t *testing.T) {
+	t.Parallel()
+
+	gogoMarshaller := &marshal.GogoProtoMarshalizer{}
+	bp, _ := NewBlockProcessor(&mock.HasherMock{}, &mock.MarshalizerMock{})
+
+	mbhr := &dataBlock.MiniBlockHeaderReserved{
+		IndexOfFirstTxProcessed: 0,
+		IndexOfLastTxProcessed:  0,
+	}
+	mbhrBytes, _ := gogoMarshaller.Marshal(mbhr)
+
+	txHash, notExecutedTxHash, invalidTxHash, rewardsTxHash, scrHash := "tx", "notExecuted", "invalid", "reward", "scr"
+	dbBlock, err := bp.PrepareBlockForDB([]byte("hash"), &dataBlock.Header{
```

Comment:
this is a bit heavy, maybe you can think of a way to refactor in a separate PR.

Reply:
I will add a task for this refactor of the tests.